config/config.exs

```diff
@@ -61,6 +61,8 @@ config :segment,
 # Configures random icon color generator
 config :code_corps, :icon_color_generator, CodeCorps.RandomIconColor.Generator
 
+config :code_corps, :corsica_log_level, [rejected: :warn]
```

Comment:
Might not be a bad idea to add a short comment to this, explaining what it does. Might be a good idea with any config setting. Maybe something like 

```
# Set Corsica logging to output a console warning when rejecting a request
# https://hexdocs.pm/corsica/Corsica.html
```

Does't cost much initially, but saves a decent amount of googling for future developers.